docs/Options.md

```diff
@@ -164,6 +164,19 @@ If you set `SPACEFISH_NODE_DEFAULT_VERSION` to the default Node.js version and y
 | `SPACEFISH_NODE_DEFAULT_VERSION` | ` ` | Node.js version to be treated as default |
 | `SPACEFISH_NODE_COLOR` | `green` | Color of Node.js section |
 
+### Docker (`docker`)
+
+Docker section is shown only in directories that contain `Dockerfile` or `docker-compose.yml` and also if the `$COMPOSE_FILE` is set.
```

Comment:
Quick note: I believe the implementation on Spaceship is that the `$COMPOSE_FILE` exists, rather than the variable being set. Just a heads up. 😄

Reply:
Damn! yes. 🙆🏻‍♂️